.github/workflows/hello-self-hosted-runner.yaml

```
@@ -1,17 +1,27 @@
+  pull_request:
+    branches:
+      - main
+#  schedule:
+#    - cron: "5 */2 * * *"
   workflow_dispatch:
 
 permissions: {}
 
 jobs:
+  docker:
+    runs-on: k8s-playground-runners
```

actionlint (1.7.7)

label "k8s-playground-runners" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-2019", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "ubuntu-20.04", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)